openslides_backend/action/actions/motion/create.py

```diff
@@ -90,6 +90,14 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 raise ActionException(
                     "You can't give amendment_paragraph_$ in this context"
                 )
+        # if lead_motion use category_id and block_id from the lead_motion
+        if instance.get("lead_motion_id"):
+            lead_motion = self.datastore.get(
```

Comment:
what happens, if the lead motion id is invalid, e.g. a non existing lead motion? Maybe add a test

Reply:
`datastore.get` will throw an error. This should be no problem

Reply:
But is it brought back to the client properly or does it do a 500er?

Reply:
It is delivered to the client